terra-batch-libchart/templates/_cbas-api.tpl

```diff
@@ -21,6 +21,13 @@ spec:
           imagePullPolicy: Always
           ports:
             - containerPort: 8080
+          resources:
```

Comment:
Do we know what the `cromwell-helm` Helm chart is used for?

It seems that this k8s template would also be pulled into `cromwell-helm` (it inherits from `terra-batch-libchart.cbas-api-deploy` as well.

It might be worthwhile following the same pattern as we have in `coa-helm/templates/cromwell.yaml` -- where these changes would only be self-contained to `coa-helm`

Reply:
No I'm afraid I don't entirely understand the directory structure of this project!  I made the cromwell changes in `coa-helm/templates/cromwell.yaml` because that was where the cromwell container specs are defined, but for cbas and wds, the yamls in `coa-helm/templates` don't define container specs but pull them in from the terra-batch-libchart templates, so that's where I made changes.

Reply:
Ah seems changes to `cromwell-helm` haven't been made in a while -- wondering perhaps if `coa-helm` is a fork or something of it....

anyways, sounds good!

Reply:
The naming is, uh, not the best (I can say that because it's largely my fault). `cromwell-helm` is the version of the chart for deploying the app in GCP/GKE, and was the original chart in this repo. `coa-helm` is for Azure/AKS.

Reply:
If it's easy enough it feels safer to override the values and leave the GCP app as-is. It's a very different deployment--every workspace gets its own GKE cluster, no idea what the specs are--and we don't want to worry about testing things there as we iterate on the Azure resources.

Reply:
Sorry trying to keep up - so instead of updating templates in `terra-batch-libchart` I should only be changing files in `coa-helm`?

Reply:
This PR sets limits for `coa-helm/templates/cromwell.yaml`, and leaves `cromwell-helm/templates/cromwell.yaml`. So I think this _is_ only addressing Azure and leaves GCP/GKE untouched. Is that correct?

Reply:
for `cromwell.yaml` yes, but for all other changes, specifically `cbas-api` in this PR (since the changes are being applied at the `terra-batch-libchart` level, it would affect cromwell-helm

Reply:
We'd probably save ourselves a headache later by being consistent with this - either set everything in the library chart or set everything in the `coa-helm` chart.

Reply:
Basically `terra-batch-libchart` contains defaults, and `cromwell-helm` (GCP) and `coa-helm` (Azure) contain overrides. The yamls are merged and the overrides take precedence.

So updating `terra-batch-libchart` would apply to both clouds (unless it's overridden); but updating coa-helm would only affect Azure.

I agree it's probably good to be consistent, and probably safer at this point to only update Azure limits, since that's the one we're really testing.